src/commands.rs

```diff
@@ -2882,6 +2882,31 @@ don't make any changes, then the operation will be aborted.",
     Ok(())
 }
 
+fn _description_template_for_cmd_split(
+    workspace_command: &WorkspaceCommandHelper,
+    intro: &str,
+    overall_commit_description: &str,
+    diff_iter: TreeDiffIterator,
+) -> Result<String, CommandError> {
+    let result1 = format!(
```

Comment:
These variable names (`result1`, `resultt2`, `jj_result2`) are not very descriptive. We can probably even remove this one by doing it at the end, i.e. with a `format!()` as the last expression in this function.